source/Mission.cpp

```diff
@@ -952,6 +966,14 @@ const MissionAction &Mission::GetAction(Trigger trigger) const
 Mission Mission::Instantiate(const PlayerInfo &player, const shared_ptr<Ship> &boardingShip) const
 {
 	Mission result;
+	std::vector<std::pair<std::string, std::string>> missionSubs;
+	for(auto &it : missionSubPhrases)
+	{
+		const Phrase *phrase = GameData::Phrases().Get(it.second);
+		missionSubs.emplace_back(it.first, phrase ? phrase->Get() : "");
```

Comment:
`phrase` is never nullptr when obtained via Set::Get. Combined with the previous change to missionSubPhrases, you can just insert the result of `Get()` (which will be `""` if the phrase was never actually loaded or defined.)